src/GitHubExtension/DeveloperId/DeveloperId.cs

```diff
@@ -67,4 +67,9 @@ public Windows.Security.Credentials.PasswordCredential RefreshDeveloperId()
         GitHubClient.Credentials = new (credential.Password);
         return credential;
     }
+
+    public Uri GetHostAddress()
+    {
+        return new Uri(new Uri(Url).GetLeftPart(UriPartial.Authority));

```

Comment:
Can this be simplified?
1. Make a Uri of Url
2. Get the left part of the authrotiy?
3. Make a Uri of that.

Reply:
The problem is GetLeftPart() returns a string, and we want to return a Uri. I could use a UriBuilder class, but it doesn't look less complicated.

Reply:
Bad input in Url can cause this to throw. Should have some try-catch here around the Uri or use TryParse and handle it with a fallback Url or make it a nullable type.

Reply:
Since this for the member variable of Uri type, the expectation is that this will not throw. Validation will be done when we create the Url member (which is coming from Octokit APIs, and should be valid).

Reply:
What are you trying to get?  What is the host address?

Reply:
This is similar to the "github.com" part, but generalized for GHES. It could literally be anything ("south.antartica.azure.co.fun") :-/

Reply:
Does Uri.Host not work?

Reply:
https://learn.microsoft.com/en-us/dotnet/api/system.uri?view=net-7.0

Reply:
Oh I realized I can just use the GitHubClient object. Will fix

Reply:
Thanks!